### DISCUSSION TITLE
Phet index page not showing after successful installation.

### DISCUSSION THREAD

**Discussion by kristianinc** [scenerystack/community#58] [Simulation Development]
Hello friends, I ran into some errors trying to install pHET. 

My homepage does not show, but instead loading the server loads the root of the folders being served.

Does anyone know how to solve this?

![phetsimsIndex](https://github.com/user-attachments/assets/77df3ddd-3272-4e69-bd7e-47524ad77d50)

**Comment by pixelzoom** [comment]
What procedure did you use to "install PhET"?

It sounds like you were expecting the installation to create a local website, with an index.html at the top level.  My understanding is that installation does not create a website, but clones a set of GitHub repositories.  Each repository is a "project", with its own .html file.  For example, under `balancing-chemical-equations/` you'll find `balancing-chemical-equations_en.html`, which you can open with your web browser.

The closest thing you'll find to a homepage is `phetmarks/index.html`.  This is a very basic user interface that is used by PhET developers to launch simulations. I don't know if there's documentation describing that user interface, but you can ask questions in the SceneryStack community.

Does that help?

**Reply by kristianinc** [reply]
That really helps so much, pixelzoom.

And it actually makes much sense now.

I appreciate it.

**Reply by kristianinc** [reply]
The procedure I used is, I cloned all the repos from the phetsims according to the development overview. Thats how I followed through and got here.

**Comment by pixelzoom** [comment]
Great - happy to help!
